src/components/addnewbook.js

```diff
@@ -1,10 +1,30 @@
+/* eslint-disable no-unused-vars */
+import { useDispatch, useSelector } from 'react-redux';
```

Comment:
- [OPTIONAL] I think you're declaring useSelector but without using it, for that reason, you needed to disable the eslint rule here. 🐱
![image](https://user-images.githubusercontent.com/20563252/131896233-aae3869f-69c7-4383-8b1d-d725f97494f8.png)